File: logica-matematica/ejercicio-006/resoluciones/lucas-pajarito/hiperdeportivos.py
autos = [
    {
        "marca": "Bugatti",
        "modelo": "Chiron",
        "velocidad_maxima": 420
    },
    {
        "marca": "Koenigsegg",
        "modelo": "Jesko",
        "velocidad_maxima": 483
    },
    {
        "marca": "Ferrari",
        "modelo": "LaFerrari",
        "velocidad_maxima": 350
    },
    {
        "marca": "McLaren",
        "modelo": "Speedtail",
        "velocidad_maxima": 403
    },
    {
        "marca": "Rimac",
        "modelo": "Nevera",
        "velocidad_maxima": 412
    }
]


def calcular_promedio_velocidad(autos):

    suma_velocidades = 0

    print("=== AUTOS HIPERDEPORTIVOS ===")

    for auto in autos:

        suma_velocidades += auto["velocidad_maxima"]

        print(f'Marca: {auto["marca"]}')
        print(f'Modelo: {auto["modelo"]}')
        print(f'Velocidad máxima: {auto["velocidad_maxima"]} km/h')
        print("------------------------------")

    promedio = suma_velocidades / len(autos)

    print(f'Promedio de velocidad máxima: {promedio:.2f} km/h')


calcular_promedio_velocidad(autos)

File: logica-matematica/ejercicio-026/resoluciones/lucas-pajarito/hiperdeportivos.js
const autos = [
    {
        modelo: "Bugatti Chiron",
        velocidadKmh: 420
    },
    {
        modelo: "Koenigsegg Jesko",
        velocidadKmh: 483
    },
    {
        modelo: "Hennessey Venom F5",
        velocidadKmh: 500
    },
    {
        modelo: "Rimac Nevera",
        velocidadKmh: 412
    },
    {
        modelo: "SSC Tuatara",
        velocidadKmh: 455
    }
];

function convertirVelocidades(autos) {

    console.log("=== AUTOS HIPERDEPORTIVOS ===");

    autos.forEach(auto => {

        const velocidadMs = auto.velocidadKmh / 3.6;
        const velocidadMph = auto.velocidadKmh * 0.621371;

        console.log(`Modelo: ${auto.modelo}`);
        console.log(`Velocidad: ${auto.velocidadKmh} km/h`);
        console.log(`Velocidad: ${velocidadMs.toFixed(2)} m/s`);
        console.log(`Velocidad: ${velocidadMph.toFixed(2)} mph`);
        console.log("------------------------------------");

    });

}

convertirVelocidades(autos);